integreat_cms/core/settings.py

```diff
@@ -83,6 +83,9 @@
 #: The time span up to which recurrent events should be returned by the api
 API_EVENTS_MAX_TIME_SPAN_DAYS = 31
 
+#: The maximum duration of an event
+MAX_EVENT_DURATION = 28
```

Comment:
```suggestion
MAX_EVENT_DURATION = int(os.environ.get("INTEGREAT_CMS_MAX_EVENT_DURATION", 28))
```
I would make this also configurable via the settings file. Otherwise we would still have to release a new version to change this value.